backend/lib/services/members.js

```diff
@@ -44,9 +44,9 @@ function fromResource (project = {}, serviceAccounts = []) {
     .chain(project)
     .get('spec.members')
     .filter(['kind', 'User'])
-    .map('name')
-    .map(username => ({
+    .map(({ name: username, role, roles }) => ({
       username,
+      roles: roles ? [role, ...roles] : [role],
```

Comment:
What if role is empty?

Reply:
gardener does not accept "empty" roles
`": ValidationError(Project.spec.members[19]): missing required field "role" in com.github.gardener.gardener.pkg.apis.core.v1beta1.ProjectMember`